app/assets/javascripts/discourse/app/routes/review-index.js

```diff
@@ -65,7 +65,7 @@ export default DiscourseRoute.extend({
       }
     });
 
-    const channel = this.currentUser.enable_redesigned_user_menu
+    const channel = this.currentUser.redesigned_user_menu_enabled
       ? `/reviewable_counts/${this.currentUser.id}`
       : "/reviewable_counts";
```

Comment:
There's no corresponding unsubscribe?

Reply:
oooh no there isn't one right now, but we definitely should have one! Sending a PR now...